components/brave_wallet/common/test_utils.cc

```diff
@@ -14,7 +14,7 @@ namespace brave_wallet {
 bool AllCoinsTested() {
   // Change hardcoded value here only when all failed callers have adequate
   // testing for newly added coin.
-  return 4 == std::size(kAllCoins);
+  return 5 == std::size(kAllCoins) && 8 == std::size(kAllKeyrings);
```

Comment:
I don't see changes in `value_conversion_utils_unittest.cc` and `account_resolver_delegate_impl_unittest.cc` but they touch AllCoinsTested

Reply:
Why there is now also keyrings check? The idea was that number of coins is enough

Reply:
Added

Reply:
> Why there is now also keyrings check? The idea was that number of coins is enough

This is an additional check to ensure keyrings list is updated along with coins list.